test/unit/api_spec.js

```diff
@@ -185,6 +185,27 @@ describe('api', function() {
         expect(stats).toEqual({ streamTypes: [], fontTypes: [] });
       });
     });
+    it('gets a raw PDF object', function() {
+      var promise = doc.getRawObject({ num: 20, gen: 0 });
+      waitsForPromiseResolved(promise, function (data) {
+        expect(data.dict.map.Length).toEqual(281);
+        expect(data.start).toEqual(1956);
+        expect(data.end).toEqual(1956+281);
+      });
+    });
+    it('gets PDF trailer object', function() {
+      var promise = doc.getRawObject('trailer');
```

Comment:
Shouldn't this be `var promise = doc.getRawObject({ ref: null });` as you have removed the 'trailer' string according to https://github.com/mozilla/pdf.js/pull/5117/files#r15894788 ?